admin/pre-reg/templates/registration_editor_templates.html

```diff
@@ -0,0 +1,143 @@
+<!-- String Types -->
```

Comment:
Consider using DTL's include tag to render these. https://docs.djangoproject.com/en/1.8/ref/templates/builtins/#include